### PR TITLE
updated gitignore to discard gradle and gradlew changes

### PR DESCRIPTION
**Pull request by parvez-amplicomm**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -9,6 +9,11 @@ target/
 *.log
 *.log.*
 
+buildSrc/gradle/
```
Comment (jgrandja) [inline .gitignore:12]: @parvez-amplicomm Why is this needed? The project never uses these 3 files. I believe this is something in your setup?

Comment (parvez-amplicomm) [reply .gitignore:12]: It's related to my local setup

Comment (jgrandja) [reply .gitignore:12]: @parvez-amplicomm Please configure your global gitignore file to include those patterns. See [gitignore](https://git-scm.com/docs/gitignore) docs for details.
